test/onevariable.jl

```diff
@@ -45,6 +45,9 @@ Base.iszero(::SymbNumber) = false
     @test axes(t) == ()
     @test axes([t]) == (Base.OneTo(1),)
 
+    @test 1 > t > t^2
```

Comment:
Could you also add a `> 0` or `>= 0` depending on what is expected?

Reply:
Done!